pages/linux/qm-snapshot.md

```diff
@@ -0,0 +1,16 @@
+# qm snapshot
+
+> Create a virtual machine snapshot.
+> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.
+
+- Create a snapshot:
+
+`qm snapshot {{vm_id}} {{snapshot_name}}`
+
+- Give a textual description or comment:
+
+`qm snapshot {{vm_id}} {{snapshot_name}} --description {{description}}`
+
+- Save the vmstate:
```

Comment:
Inside the snapshot?
```suggestion
- Create a snapshot including the vmstate:
```
idk what is vmstate actually

Reply:
[here's a good explanation](https://wiki.openstack.org/wiki/VMState)